lib/URLButton.js

```diff
@@ -0,0 +1,130 @@
+//reimplimentation of add-on sdk button/toggle, found at
```

Comment:
Right now I'm just reimplementing pageAction's ability to place a button in the nav bar, and attaching an sdk panel to that button.